kafka-ui-serde-api/pom.xml

```diff
@@ -13,10 +13,12 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-install-plugin</artifactId>
-                <version>2.5.2</version>
+                <version>2.4</version>
```

Comment:
any specific reason we're downgrading?

Reply:
@Haarolean Outdated local `master` branch. 🤦 

I've updated the PR.